docker/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+if [[ "$1" = "serve" ]]; then
+    shift 1
+    mxnet-model-server --start "$@"
+else
+    eval "$@"
```

Comment:
When do we get into this else?

Reply:
for example:
1. docker run -itd mms /bin/bash
2. docker run -itd mms mxnet-model-server --models ALL ...

Reply:
Oh. Looks like we would need to run the whole command anyways. Then we don't need entry-point.  What are we saving by adding this entry point?

Reply:
The one passed in is CMD, not entrypoint, to override entrypoint user has to use --entrypoint, which is not most user used to do.
The convention is use a fixed entrypoint named dockerd-entrypoint and use CMD as parameter. So it looks like you can run any commands, actually they just pass parameters to entrypoint.

Reply:
I see what's happening. I think you are using CMD as a default value to run the container. This looks good.